Samples/excel-insert-file/README.md

```diff
@@ -0,0 +1,177 @@
+---
+page_type: sample
+products:
+- office-excel
+- office
+languages:
+- javascript
+extensions:
+  contentType: samples
+  technologies:
+  - Add-ins
+  createdDate: "5/18/2022 10:00:00 AM"
+description: "This sample shows how to insert a template from an external Excel file and populate it with JSON data."
+---
+
+# Insert an external Excel file and populate it with JSON data
+
+## Summary
+
+This sample shows how to insert an existing template from an external Excel file into the currently open Excel file. Then it retrieves data from a JSON web service and populates the template for the customer.
+
+> **Note:** The features used in this sample are currently in preview and subject to change. They are not currently supported for use in production environments. To try the preview features, you'll need to [join Office Insider](https://insider.office.com/join). A good way to try out preview features is to sign up for an Office 365 subscription. If you don't already have an Office 365 subscription, get one by joining the [Office 365 Developer Program](https://developer.microsoft.com/office/dev-program).
```

Comment:
```suggestion
> [!NOTE]
> The features used in this sample are currently in preview and subject to change. They are not currently supported for use in production environments. To try the preview features, you'll need to [join Office Insider](https://insider.office.com/join). A good way to try out preview features is to sign up for an Office subscription. If you don't already have an Office subscription, get one by joining the [Microsoft 365 Developer Program](https://developer.microsoft.com/office/dev-program).
```

Reply:
@lindalu-MSFT This readme is only in the repo and not published to docs. So we don't use the docs markdown tags.

Reply:
Oh yeah, I figured that out but forgot to remove this suggestions! I'll keep a better eye out what is YAML and what is md.